homeassistant/util/color.py

```diff
@@ -320,12 +338,24 @@ def color_hsv_to_RGB(iH: float, iS: float, iV: float) -> Tuple[int, int, int]:
 
 
 # pylint: disable=invalid-sequence-index
-def color_xy_to_hs(vX: float, vY: float) -> Tuple[int, int]:
+def color_hs_to_RGB(iH: float, iS: float) -> Tuple[int, int, int]:
+    """Convert an hsv color into its rgb representation."""
+    return color_hsv_to_RGB(iH, iS, 100)
```

Comment:
This still seems dangerous to me. It makes it easy for light platforms to ignore the brightness issue and that will often be wrong for lights where the API uses RGB. The result is that `hs_color` ends up modifying the brightness.

I would like to force platforms to use `color_hsv_to_rgb()` and have them provide a dummy value if brightness really does not matter in their case.